src/RestaurantButton.js

```diff
@@ -0,0 +1,8 @@
+import React, { useState } from "react";
+
+const RestaurantButton = (props) => {
+    return 
+    <button className="btn btn-primary" onClick={props.order}>Add</button>
```

Comment:
Could we put the button on the same line as our return statment? Or we could wrap the button in brackets ().

It looks a bit like this is returning nothing otherwise!

Reply:
yes, can do that.